### DISCUSSION TITLE
Media controls on preview

### DISCUSSION THREAD

**Discussion by hardcodedvalue** [ciderapp/Cider-2#342] [Feature Requests]
On windows, I sometimes use the media controls in the app thumbnail to stop/start songs. It would be nice to have that enabled.

Example: 

Spotify:
![image](https://github.com/ciderapp/Cider-2/assets/78376557/4c8f0d09-e2d1-4013-86f8-37936fb47929)

Ciderapp:
![image](https://github.com/ciderapp/Cider-2/assets/78376557/6bcf8190-adb3-42ab-85dd-78f72c79c765)

**Comment by JugoBonito** [comment]
That would be an awsome feature
